pkgs/by-name/cl/clang-uml/package.nix

```diff
@@ -1,12 +1,11 @@
 {
   lib,
   fetchFromGitHub,
-  stdenv,
+  clangStdenv,
```

Comment:
probably better to do this override in all-packages.nix https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults